unfoldingWord/translationCore
# Removed static folder and added resources submodule

#### This pull request addresses:
This PR removes the old static folder and brings in the resources as a self contained module


#### How to test this pull request:
Make sure everything in the app works the same as far as data is concerned (tN, tW)
Note: The scripture pane is not functional due to another issue on develop

Also make sure that the submodule looks as expected, folder structure/configuration wise


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2630)
